SPECS-EXTENDED/perl-Net-LibIDN2/perl-Net-LibIDN2.spec

```diff
@@ -61,7 +85,14 @@ find $RPM_BUILD_ROOT -type f -name '*.bs' -size 0 -delete
 %{perl_vendorarch}/Net*
 %{_mandir}/man3/*
 
+%files tests
+%{_libexecdir}/%{name}
+
 %changelog
+* Tue Dec 10 2024 Jyoti Kanase <v-jykanase@microsoft.com> - 1.02-1
```

Comment:
Same goes here, increase the release number to 9.

Reply:
I had a chat with the author of this PR. Since it is just an upgrade no need to increase the release number.